src/components/HeaderComponent.vue

```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+const createTodo = () => {
+  console.log('Create todo...')
+}
+</script>
+
+<template>
+  <div class="flex items-center justify-between">
+    <h1 class="font-semibold text-7xl">To do list</h1>
+    <button @click="createTodo">
+      <img src="../assets/svg/add-button.svg" />
```

Comment:
Use [Heroicons](https://github.com/tailwindlabs/heroicons) please, you can search the plus icon from [here](https://heroicons.com/)